assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/inlineStyles.ts

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
This is also temporary?

Reply:
yes we will redo all of this when we work on the presentation layer